prediction_market_agent/agents/think_thoroughly_agent/think_thoroughly_agent.py

```diff
@@ -514,7 +502,7 @@ def generate_prediction_for_one_outcome(
             original_question=original_question,
             p_yes=prediction.outcome_prediction.p_yes,
             confidence=prediction.outcome_prediction.confidence,
-            reasoning=prediction.outcome_prediction.reasoning,  # TODO: Possible improvement: Prophet currently doesn't return reasoning of its prediction, so it's just None all the time.
+            reasoning=prediction.outcome_prediction.reasoning,
```

Comment:
It returns it now, thanks to Evan! Added `include_reasoning=True,` above in the `prophet_make_prediction` call.